config.rb

```diff
@@ -79,6 +79,32 @@
   },
 ]
 
+explorer_events = [
+  {
+    slug: 'evento-1',
+    date: '2019-07-12',
+    title: 'Evento 1',
+    icon_path: 'expo-icon.jpg',
+    category: 'Esploratori',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+  },
+  {
+    slug: 'evento-2',
+    date: '2019-07-16',
+    title: 'Evento 2',
+    icon_path: 'gen-icon.png',
+    category: 'Genitorialità',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+  },
+  slug: 'evento-2',
```

Comment:
@EnigmaEdivad Qua manca il `{` di apertura.
E in seguito l'indentazione corretta dei dati di questo evento.
Rinominalo poi in "Evento 3".